### DISCUSSION TITLE
AR.j team

### DISCUSSION THREAD

**Discussion by kalwalt** [AR-js-org/AR.js#533] [Team Posts]
The **AR.js** team will be our place where discuss and plan the future of AR.js.
I invited also you @nicolocarpignoli because seems correct, even if you are not active anymore, to e part of. @nickw1 and all in the org: i will invite other members in the team just let me know who is motivated to enter.

**Comment by nicolocarpignoli** [comment]
Hi there, and thanks Walter for the invitation :) 
As I wrote before on the thread, I leave to you any decisions, but the only thing that I don't like (in case) is the suppression of AR.js, which has no sense to me. 
Anyway, I have no fear for this as I feel that you too do not want that.

Maybe it's okay to be realistic about the difficulty in working on the AR.js current codebase. AR.js was, from its beginning, a project used by a lot of people but where very few contributed (us, Jerome Etienne, few others).
This is likely to remain this way, in my opinion, so it will be inevitable to look for "external" help, which might mean merging with other libraries (like mind ar js).
I left to you this decision and how to deal with the mind ar js creator. 

If you need me, I'm here!

**Comment by kalwalt** [comment]
Hi @nicolocarpignoli thanks for accepting the invite!
> As I wrote before on the thread, I leave to you any decisions, but the only thing that I don't like (in case) is the suppression of AR.js, which has no sense to me.
Anyway, I have no fear for this as I feel that you too do not want that.

No suppression will not happens for sure, not by me!
AR.js has a lot of defects but i think can be improved a lot, but as you said there are few people that contribute and contributed. Many many people that open issue but very few that send PR or give other advice.
Enlarge the codebase with eventuallly mind-ar.js is a possibility, though i think im not considering to drastically changes (well, with the ES6 feature many things will change)
A good strategy would find other developers that want to be engaged in the open source. Yesterday  i was contacted by @andypotato for AR.js, It will be nice if you Andreas can help us again!

**Comment by andypotato** [comment]
Hey guys,

haven't been in the loop for a while due to personal reasons. If there's any way for me to help then please let me know!

Cheers,
Andreas

**Comment by kalwalt** [comment]
I created a Discord channel where we can discuss the future of Ar,js if you are interested you can join us at https://discord.gg/Y82hMdkN

**Comment by nickw1** [comment]
@kalwalt many thanks for this! Sorry for late reply, I've been busy with work this week.
